.github/workflows/go.yml

```diff
@@ -0,0 +1,32 @@
+name: Go
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Go
+      uses: actions/setup-go@v2.1.3
+      with:
+        go-version: 1.16.7
```

Comment:
Is this specific version required? We've generally found it a bit better just to have `go-version: 1.x` here I think in order to avoid having to add new versions.

Reply:
done